src/main/resources/templates/hremployee/bootcamps.html

```diff
@@ -93,7 +93,7 @@ <h1>Running Bootcamps</h1>
 
 
 
-<div class="ui segment">
+<div class="ui segment" style="margin-left: 25px; margin-right: 25px;">
```

Comment:
wanneer mogelijk, inline css in een apart css bestand zetten (opmaak kan dan gedeelt worden met meerdere tags)